src/Faker/Provider/bs_BA/Address.php

```diff
@@ -0,0 +1,319 @@
+<?php
+
+namespace Faker\Provider\bs_BA;
+
+class Address extends \Faker\Provider\Address
+{
+    protected static $streets = array(
```

Comment:
Can you add a datasource? Can you order alphabetically, one line per first letter?